telemetry/about-telemetry.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="about-telemetry"]
+= About Telemetry
+include::modules/common-attributes.adoc[]
+:context: about-telemetry
+
+toc::[]
+
+
+[role="_abstract"]
+{rh-rhacs-first} collects anonymized aggregated information about product usage and product configurations.
+This information can help Red Hat to better understand how customers use the product and which areas to prioritize for improvements.
+Red Hat uses this information to improve the user experience.
```

Comment:
**nitpick:** I usually avoid split infinitives, however when I break it down, I think we could write a simpler sentence. Maybe 
```
It helps Red Hat understand how everyone uses the product and identify areas 
to prioritize for improvements. In addition, Red Hat uses this information to 
improve the user experience.
```